From bc3694b178db66406f119e1440bff305238a9149 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 10 May 2023 13:39:25 +0200
Subject: [PATCH 0105/1141] arm64: defconfig: enable STM32 DMA3 in addons
 fragment

Enable STM32 DMA3 support on STM32 arm64 platforms in addons fragment.

Change-Id: Iae2fb7787220d162cab2a4c8de5055aa784300e8
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/305218
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -6,3 +6,4 @@ CONFIG_DYNAMIC_DEBUG=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
+CONFIG_STM32_DMA3=y
